bootcamp/tutorials/integration/integrate_with_phidata.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<p>To ingest data into the knowledge base, create an instance of <code>PDFUrlKnowledgeBase</code> by specifying the <code>urls</code> (here we use an example PDF) and the Milvus <code>vector_db</code> object created above.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>